Fix filter height

rockingskier opened the pull request:
Add `div` around the filter to breaks it out of the `flexbox` position/size.  This lets it keep a static size.

Commit:
Re-add `flex` proprety to ensure the filter keeps it's height

alexkuz commented:
Hm, I thought it was fixed already... Anyway, I believe it's better to use `flex: 0 0 auto;` instead of creating another container.

rockingskier commented:
Yes, the previous fix removed the `flex` property which made it work better but still resulted in a squashed filter box.

My flex knowledge is limited at best so I added the `div`.

Anyway, updated to just re-implement the `flex` property.

alexkuz commented:
Published in `v0.6.1`
